chainerx_build_helper.py

```diff
@@ -40,6 +41,11 @@ def run(self):
         if cmake_version < '3.1.0':
             raise RuntimeError('CMake >= 3.1.0 is required to build ChainerX')
 
+        generator = os.getenv('CHAINERX_CMAKE_GENERATOR', None).lower()
```

Comment:
You cannot chain it with `lower()` as `getenv` might return `None`. Could you separate the statements with a `None` check?

Reply:
what about default to `''`, an empty string. Will be more straight forward